src/pages/publication/files_and_license_tab/FileCard.tsx

```diff
@@ -111,7 +120,25 @@ const FileCard: React.FC<FileCardProps> = ({ file, removeFile, updateFile }) =>
           </StyledFormControl>
 
           <StyledFormControl>
-            <TextField select variant="outlined" label={t('files_and_license.license')}></TextField>
+            <StyledSelect
+              select
+              variant="outlined"
+              defaultValue=""
+              label={t('files_and_license.license')}
+              onChange={({ target: { value } }) => {
+                updateFile({
+                  ...file,
+                  license: value,
+                });
+              }}>
+              {licenses.map((license: License) => (
+                <MenuItem key={license.name} value={license.name} divider dense>
+                  <ListItemIcon>
+                    <img src={license.image} alt={license.name} />
```

Comment:
bruk gjerne Avatar (brukt i Modal.tsx) i stedet for direkte img, slik at vi får standardisert bruken av ikoner.

Reply:
Ser ut som at Avatar er lagd for ikoner/bilder som skal vises med høyde = bredde. Det fins en `ListItemAvatar` med samme begrensning også

Reply:
Altså: bildet blir klippet til enten sirkel eller kvadrat, men vi trenger rektangel. Finner ikke noen komponent for dette